panda/src/speedtree/speedTreeNode.I

```diff
@@ -47,7 +47,6 @@ get_tree(int n) const {
  */
 INLINE const SpeedTreeNode::InstanceList &SpeedTreeNode::
 get_instance_list(int n) const {
-  nassertr(n >= 0 && n < (int)_trees.size(), *(InstanceList *)NULL);
```

Comment:
Resolution: `assert` (and add TODO comment saying this needs to be nassertr but there's no good, safe, return type so we have to use assert for now)

Reply:
OK, though I don't know if anyone is going to ever be able to compile this code, so I'm not so worried about it.

FWIW I think we should also try to replace int with size_t in methods like this so that we get less ugly asserts, but that is a separate issue.